pdf_on_submit/pdf_on_submit/doctype/pdf_on_submit_settings/pdf_on_submit_settings.json

```
@@ -31,12 +32,17 @@
    "fieldtype": "Table",
    "label": "Enabled For",
    "options": "Enabled DocType"
+  },
+  {
+   "fieldname": "timeout",
+   "fieldtype": "Int",
+   "label": "Timeout"
   }
```

```suggestion
  {
   "fieldname": "timeout_seconds",
   "fieldtype": "Int",
   "label": "Timeout (Seconds)"
}
```

Die Info zur Einheit wäre sicher auch noch ganz nützlich